Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM python:3.9.7-buster as base
+
+
+RUN mkdir -p /app/src
+WORKDIR /app/src
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+ENV PATH=$PATH:/root/.poetry/bin 
+ENV FLASK_APP=todo_app/app
+
+COPY poetry.lock poetry.toml pyproject.toml ./ 
+
+RUN poetry install --no-dev
+COPY todo_app ./todo_app
+
+EXPOSE 5000
+
+FROM base as production
+EXPOSE 8000
+
+ENV FLASK_ENV=production
+ENTRYPOINT ["poetry","run","gunicorn","--bind","0.0.0.0:8000","todo_app.app:create_app()"]
+
+FROM base as development
+
+ENTRYPOINT ["poetry","run","flask","run","--host","0.0.0.0"] 
+
+
+# These are the commands to build the image and run docker
```

Comment:
These instructions are in your README (as they should be), so you don't need them here as well. Having two copies of instructions just means one is going to end up out of date.